genienlp/run_generation.py

```diff
@@ -152,8 +152,8 @@ def sample_sequence(model, length, context, position_ids, num_samples=1, tempera
                 inputs['past'] = past
                 if past is not None:
                     inputs['input_ids'] = next_token
-                    inputs['position_ids'] = position_ids[:, next_index-1]
-                    inputs['token_type_ids'] = segment_ids[:, next_index-1]
+                    inputs['position_ids'] = position_ids[:, next_index-1].unsqueeze(-1)
+                    inputs['token_type_ids'] = segment_ids[:, next_index-1].unsqueeze(-1)
```

Comment:
This was not "Remove unused code". Sneaked in by accident?

Reply:
It failed one of the paraphrasing tests with the newer transformer version, so I had to make this change. All this code will be obsolete when we merge `wip/paraphrase` branch.